client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -58,8 +57,7 @@ class Upload extends React.Component {
 		installing: React.PropTypes.bool,
 	};
 
-	constructor( props ) {
-		super( props );
+	componentDidMount() {
```

Comment:
I don't see any issue here but I'm curious about what prompted this change?
Was the `clearThemeUpload` action not getting called at the right time?